<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据同步配置</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QFrame" name="frame_4">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>同步信息配置</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>身份选择</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_character">
           <item>
            <property name="text">
             <string>源端</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>目的端</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>同步策略</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_mode">
           <item>
            <property name="text">
             <string>订阅发布</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>配置发布</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>同步时机</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_time">
           <item>
            <property name="text">
             <string>立即同步</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5分钟后同步</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10分钟后同步</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>数据同步频率</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_freq">
           <item>
            <property name="text">
             <string>1秒</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5秒</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>30秒</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_map">
         <property name="text">
          <string>测试使用：目的端模式名.表名。 e.g.SYSDBA.T2(注意！先填写本栏再进行详细信息配置)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_test">
         <property name="text">
          <string>详细信息配置</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>本地数据库连接</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>数据库地址:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_ip">
           <property name="text">
            <string>localhost</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>数据库端口:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_port">
           <property name="text">
            <string>5236</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>数据库名称:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_dbname">
           <property name="text">
            <string>DM8</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>账号:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_username"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>密码:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_password">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="text">
            <string>连接</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>数据传输配置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSYN">
        <property name="text">
         <string>数据同步</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>同步任务日志</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>错误日志</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>同步任务描述</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>任务名称：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_mission_name">
           <property name="text">
            <string>同步任务名称</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>任务描述：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_mission_dsp">
           <property name="text">
            <string>同步任务简述</string>
           </property>
           <property name="cursorPosition">
            <number>6</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton_condition_monitor">
      <property name="text">
       <string>同步状态监控</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
